truthy/truthy.go

```diff
@@ -0,0 +1,27 @@
+// Package truthy implements helpers to test the truthy-ness of the values.
+package truthy
+
+import "strings"
+
+var nos = [...]string{
```

Comment:
hmmm, what do you think should happen if someone provides "VAR=fooo123" -- should this be true? i guess it can be

Reply:
sgtm